caro-odk.yaml

```diff
@@ -0,0 +1,50 @@
+id: caro
```

Comment:
I love this, bravo.

Reply:
But do not check in caro-odk.yaml, you should only check in `src/ontology/caro-odk.yaml`